README.md

```diff
@@ -3,43 +3,46 @@
 [![Join the chat at https://gitter.im/fossasia/yaydoc](https://badges.gitter.im/fossasia/yaydoc.svg)](https://gitter.im/fossasia/yaydoc?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 > Docs! Yay!
-# Usage
-- generate github personal token by following this guide (https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/)
-- set the following environment variable in the travis       
-     1) AUTHOR      (author of the repository)
-     2) DOCPATH     (path of the documentation)
-     3) GITURL      (https url not ssh)
-     4) PROJECTNAME (project name)
-     5) VERSION     (version of project)
-     6) DOCURL      (custom url at which site should be published.
-                     If it is not set then default gh-pages url is used which is of the format
-                     <username or organization>.github.io/<reponame>)
-     7) DOCTHEME    (name of the theme, default alabaster)
-
- Click [here](http://www.sphinx-doc.org/en/stable/theming.html#builtin-themes) to see a list of builtin themes.
- You can also specify any custom themes available on pypi.
-
- your git url should look like this
-  ```shell
-  https://<username>:<token>@github.com/<organisation or username>/<repname>
-  ```
-- add the following to travis.yml
-
-If language is python
 
+### Prerequisites
+- Create a directory in your repository containing all the markup files along with an `index.rst` file which contains [toctrees](http://www.sphinx-doc.org/en/stable/markup/toctree.html) to link the various documents.
+- Create a `gh-pages` branch in your repository.
+- Register your repository with [Travis CI](https://www.google.co.in/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0ahUKEwjn8Y3v1_zTAhUKtI8KHaJWBssQFgg6MAA&url=https%3A%2F%2Ftravis-ci.org%2F&usg=AFQjCNFI_4340tr1MDD78A2HC-luYCxgAQ&sig2=d-UEeLAfCHrO-7wo9qYSGg).
+
+## Usage
+- Generate `Github Personal Token` by following this [Guide](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/)
+- Set the following Environment Variables in Travis CI. [Guide](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings)     
+
+| Environment Variable | Description                                       | Default / FORMAT  |
+|:--------------------:| ------------------------------------------------- |:-----------------:|
+| AUTHOR               | Author of the repository.                         | NONE * |
+| DOCPATH              | Path of the documentation.                        | NONE * (eg. `docs/`) |
+| DOCTHEME             | Name of the theme.                                | alabaster ([built-in themes](http://www.sphinx-doc.org/en/stable/theming.html#builtin-themes)) | 
```

Comment:
No list as such. Specifying that any custom theme from pypi is supported should be enough.